lib/liquid/context.rb

```diff
@@ -47,6 +48,14 @@ def add_filters(filters)
       @strainer = nil
     end
 
+    def add_global_filters(global_filters)
+      @global_filters = global_filters if global_filters.respond_to?(:call)
```

Comment:
Agreed.  You should either raise an `::ArgumentError` if the argument doesn't respond to `:call` or remove the check entirely.